PowerEditor/src/NppIO.cpp

```diff
@@ -93,7 +94,12 @@ DWORD WINAPI Notepad_plus::monitorFileOnChange(void * params)
 					{
 						if (dwAction == FILE_ACTION_MODIFIED)
 						{
-							::PostMessage(h, NPPM_INTERNAL_RELOADSCROLLTOEND, reinterpret_cast<WPARAM>(buf), 0);
+							if (!bDoneOnce)

```

Comment:
What's the point of this boolean? It'll be always true this condition.

Reply:
Please see my changes in context by looking at the whole case statement [here](https://github.com/BubuC/notepad-plus-plus/blob/cf99b9ec59a20129f6093a807896600f16ff958d/PowerEditor/src/NppIO.cpp#L79)

The boolean is set to false outside the loop, and set to true once inside, so that only first of a flood of matching events triggers PostMessage(h, NPPM_INTERNAL_RELOADSCROLLTOEND...